src/clue/adafruit_clue.py

```diff
@@ -279,6 +284,7 @@ def acceleration(self):
           while True:
             print("Accel: {:.2f} {:.2f} {:.2f}".format(*clue.acceleration))
         """
+        telemetry_py.send_telemetry(TelemetryEvent.CLUE_API_ACCELERATION)

```

Comment:
It seems that `CLUE_API_ACCELERATION` was not defined in the enum.

Reply:
fixed